test/index.js

```diff
@@ -0,0 +1,176 @@
+  });
+
+  it('posts a snooze comment - no label', async () => {
+    commentEvent.payload.comment.body = '@probot, we should snooze this for a while, until July 1, 2018 13:30';
+    await robot.receive(commentEvent);
+    expect(github.repos.getContent).toHaveBeenCalledWith({
+      owner: 'baxterthehacker',
+      repo: 'public-repo',
+      path: '.github/probot-snooze.yml'
+    });
+    expect(github.issues.edit({
+      number:2,
+      owner: 'baxterthehacker',
+      repo: 'public-repo',
+      state: 'closed',
+      labels:[{
+        url: 'https://api.github.com/repos/baxterthehacker/public-repo/labels/bug',
+        name: 'bug',
+        color: 'fc2929'
+      },
+        'probot:freeze']
+    }));
+    expect(github.issues.createComment).toHaveBeenCalledWith({
+      number:2,
+      owner: 'baxterthehacker',
+      repo: 'public-repo',
+      body: 'Sure thing. I\'ll close this issue for a bit. I\'ll ping you around 07/01/2018 :clock1: ' +
+        '<!-- {"assignee":"baxterthehacker","unfreezeMoment":"2018-07-01T17:30:00.000Z","message":"Hey, we\'re back awake!"}-->'
+    });
+  });
+
+  it('posts a snooze comment - with label', async () => {
+    commentEvent.payload.comment.body = '@probot, we should snooze this for a while, until July 1, 2018 13:30';
+    commentEvent.payload.issue.labels.push({
+      url: 'https://api.github.com/repos/baxterthehacker/public-repo/labels/probot:freeze',
+      name: 'probot:freeze',
+      color: 'cccccc'
+    });
+    await robot.receive(commentEvent);
+    expect(github.repos.getContent).toHaveBeenCalledWith({
+      owner: 'baxterthehacker',
+      repo: 'public-repo',
+      path: '.github/probot-snooze.yml'
+    });
+    expect(github.issues.edit({
+      number:2,
+      owner: 'baxterthehacker',
+      repo: 'public-repo',
+      state: 'closed',
+      labels:[{
+        url: 'https://api.github.com/repos/baxterthehacker/public-repo/labels/bug',
+        name: 'bug',
+        color: 'fc2929'
+      },
+        'probot:freeze']
+    }));
+    expect(github.issues.createComment).toHaveBeenCalledWith({
+      number:2,
+      owner: 'baxterthehacker',
+      repo: 'public-repo',
+      body: 'Sure thing. I\'ll close this issue for a bit. I\'ll ping you around 07/01/2018 :clock1: ' +
+        '<!-- {"assignee":"baxterthehacker","unfreezeMoment":"2018-07-01T17:30:00.000Z","message":"Hey, we\'re back awake!"}-->'
+    });
+  });
+
+  it('test visitor activation', async () => {
+    await robot.receive({
+      event: 'test',
+      payload: {
+        action: 'visit',
+        repository: {
+          owner: {
+            login:'baxterthehacker'
+          },
+          name:'public-repo'
+        },
+        installation: {
+          id: 13055
+        }}});
+    expect(github.repos.getContent).toHaveBeenCalledWith({
+      owner: 'baxterthehacker',
+      repo: 'public-repo',
+      path: '.github/probot-snooze.yml'
+    });
+
+    // Expect(github.issues.edit).toHaveBeenCalledWith({
```

Comment:
these expects never resolve, even though i console.log the contents of the github calls and they match exactly. My only thought is that somehow the `github` object that we are expecting is not the `github` object that is being used to make the requests (some sort of pass by value/pass by reference issue)?

cc/ @bkeepers